### ISSUE TITLE
Pumpkins grow in straight lines, fruitless

### ISSUE THREAD

**Issue by Heptagon-ru** [anegostudios/VintageStory-Issues#40]
1.5.2.2, official server.
Twice my pumpkin grew into a cross of long straight wines, no fruits. Was grown on the same place, first time on medium soil, last on rich.
Tests in creative mode with time speed 6000-9000 didnt show such anomaly.
![2018-03-22_10-09-56](https://user-images.githubusercontent.com/25122345/37862511-ebf485d6-2f5e-11e8-828d-899c26915559.png)
![2018-03-24_12-02-29](https://user-images.githubusercontent.com/25122345/37862512-ec15dca4-2f5e-11e8-9933-edba409844dc.png)

**Comment by milochristiansen** [comment]
I'm pretty sure I had the same issue in my patch on the official server. No fruit, and long straight vines. The patch doesn't really allow the lengths shown in the above screenshots though, so at the time I though I was just unlucky and went on my way (I was in something of a hurry so I didn't examine the plants too closely either).

**Comment by Heptagon-ru** [comment]
In 1.5.2.3 fruits appeared right away on near the flowering wine.
![2018-03-24_22-30-07](https://user-images.githubusercontent.com/25122345/37868129-eebf8f5a-2fb2-11e8-8cce-e957378e0ad3.png)

**Comment by Heptagon-ru** [comment]
Sorry, in 1.5.2.6 behavor still funky. Was planted on 1.5.2.6, returned after ~2 irl days.
![2018-03-31_00-19-33](https://user-images.githubusercontent.com/25122345/38154193-6fd93314-3479-11e8-9a60-4b773616a1a8.png)
Please reopen the tickei

**Comment by Heptagon-ru** [comment]
Being a early duplicate of #63, closing this too after #63.
